### PR TITLE
Removed 2nd example of patch usage

### PR DESCRIPTION
**Pull request by rainabba**
Sorry for the PR mess. Still getting the hang of this. Looks like I should have branched before adding the feature (after README.md change) and then I could have done a compare with a seperate branch. Below find what should have been the info for 2 different PRs.

docs(grunt-bump): Removes a potentially confusing and redundant example in README.md

feat(grunt-bump): Adds commitmsg option to allow for appending commit messages at the command line

### CODE REVIEW

**Comment by rainabba** [comment]
Just found the PR guidelines. Sorry for not following for this one.

**Comment by rainabba** [comment]
reopened because I'm still figuring out PRs. See notes in original commented which have been updated.

**Comment by eddiemonge** [comment]
The commit messages dont follow the guidelines in the contributing.md file

File: README.md [hunk 1]
```diff
@@ -157,12 +157,6 @@ $ grunt bump:prerelease
 >> Tagged as "v1.0.0-1"
 >> Pushed to origin
 
-$ grunt bump:patch
```
Comment (eddiemonge) [inline README.md]: why did you remove this part?

File: README.md [hunk 2]
```diff
@@ -180,6 +174,19 @@ $ grunt bump --setversion=2.0.1
 >> Pushed to origin
 ```
 
+If you want to append commit message content, you can use the ```commitmsg``` tag in the command line.
```
Comment (eddiemonge) [inline README.md]: the commit msg should be about the release. the commit messages on the commits leading up to the release should be about whats in it. Those ideally should get put into some kind of changelog.

File: tasks/bump.js [hunk 3]
```diff
@@ -56,6 +56,16 @@ module.exports = function(grunt) {
         exactVersionToSet = false;
     }
 
+
+    //Use: --commitmsg="add me to commit message"
+    var appendToCommitMessage = grunt.option('commitmsg');
```
Comment (eddiemonge) [inline tasks/bump.js]: if you are going to have a flag like `commitmsg` it should change the commit message, not append to it.

File: tasks/bump.js [hunk 4]
```diff
@@ -176,7 +186,7 @@ module.exports = function(grunt) {
 
 
     // PUSH CHANGES
-    runIf(opts.push, function() {
+    runIf( opts.push, function() {
```
Comment (eddiemonge) [inline tasks/bump.js]: none of the other `runIf`s have the first space

**Comment by eddiemonge** [comment]
I'm inclined to close this:

> the commit msg should be about the release. the commit messages on the commits leading up to the release should be about whats in it. Those ideally should get put into some kind of changelog.

**Comment by rainabba** [comment]
Your call. As I said, I'm working on getting to understand this system.

I did see the discussion at #111 as I was researching for another issue I was thinking about addressing, but I'd already finished my code and the PR before seeing that and realizing what it was.

Close it if that's your decision, but that sort of comment and response doesn't encourage me (or anyone else) to contribute further.

File: tasks/bump.js (hunk 4 above)
Comment (rainabba) [inline tasks/bump.js]: Okay?

File: tasks/bump.js (hunk 3 above)
Comment (rainabba) [inline tasks/bump.js]: Except that there's already an option for that and I'm adding functionality, not replacing it. I agreed with the v1.2.3 approach and didn't see any reason to override it or force someone to choose between one commit message approach over the other. Personally, I just wanted to add detail to my commit message.

File: README.md (hunk 1 above)
Comment (rainabba) [inline README.md]: "Removes a potentially confusing and redundant example in README.md"

bump:patch is already covered quite nicely above. I spent a couple minutes trying to figure out what that part of the README was explaining and all I could come up with is that it was a duplicate of the example for bump:patch above.

Comment (eddiemonge) [inline README.md]: it was meant to show the prerelease version being bumped to a patch version, in order to demonstrate the git version bump. It might be more clear to add comments to what is happening if its not clear already

Comment (rainabba) [inline README.md]: Yes, it might. Especially by someone who knows what was intended :)

Comment (eddiemonge) [inline README.md]: can you add the comments (in a separate PR) and unremove those lines in this PR?

File: tasks/bump.js (hunk 3 above)
Comment (eddiemonge) [inline tasks/bump.js]: maybe the commitMsg should accept a template-able string with the version number, if it already doesnt

File: tasks/bump.js (hunk 4 above)
Comment (eddiemonge) [inline tasks/bump.js]: it adds noise to the commit to have things like this added in. commits should be small and clean

**Comment by eddiemonge** [comment]
Closing to let @rainabba clean up in separate commits.
